docs/generated/errors-list.md

```diff
@@ -109,6 +109,13 @@ The `galasactl` tool can generate the following errors:
 - GAL1106E: Could not get security bearer token from API server. Reason: '{}'. Please ensure you have allocated a personal access token and configured your client program by storing it in your galasactl.properties file together with the related client ID and secret
 - GAL1107E: Could not get security bearer token from file '{}'. Reason: '{}'. Please ensure you are authenticated by running 'galasactl auth login' and that your personal access token has not expired or been revoked
 - GAL1108E: Invalid bearer token. Please ensure you are authenticated by running 'galasactl auth login' and that your personal access token has not expired or been revoked
+- GAL1109E: Unable to access the file. Reason: '{}'
+- GAL1110E: Could not get the content of the yaml file. Reason: '{}'
+- GAL1111E: Error unmarshalling yaml. Invalid yaml. Reason: '{}'. Invalid yaml:'{}'
+- GAL1112E: Error unmarshalling into json. Invalid input. Reason: '{}'
+- GAL1113E: Response status code is not okay (200 or 201). Reason: '{}'
```

Comment:
If I as a galasactl user saw that I wouldn't have a clue what that meant.
Errors like this need to be in the log, but not in front of the user.

Relate error messages to what the user has just done to get the error, and say what they need to do next. Not easy I know...

So perhaps something more depending on the error code ? 

4xx-type errors: `Failure reported by the Galasa Ecosystem. The Ecosystem believes there is a problem with this client program. This could indicate that the client and Ecosystem have a mis-match in their versions, and the client needs to be upgraded, or downgraded to match. Use the `galasactl version` to find the version of this tool.

Or for 500/5xx codes: `A failure was reported from the Galasa Ecosystem where the Ecosystem is at fault. Please contact your Galasa Ecosystem adminstrator. Reason: {}`